plugins/uefi-capsule/fu-uefi-device.c

```diff
@@ -739,6 +739,7 @@ fu_uefi_device_init(FuUefiDevice *self)
 {
 	fu_device_set_summary(FU_DEVICE(self), "UEFI ESRT device");
 	fu_device_add_protocol(FU_DEVICE(self), "org.uefi.capsule");
+	fu_device_add_internal_flag(FU_DEVICE(self), FU_DEVICE_INTERNAL_FLAG_NO_LID_CLOSED);
```

Comment:
I think you need to check with all the OEMs on LVFS before applying this to everyone. It may only make sense for certain ones.

Reply:
I've restricted this to Lenovo as it's this that does entirely the wrong thing when the lid is closed. I'll check with Crag and the HP guys.